examples/stress_tests/many_animated_sprites.rs

```diff
@@ -6,7 +6,7 @@
 //! Having sprites out of the camera's field of view should also help stress
 //! test any future potential 2d frustum culling implementation.
 
-use std::time::Duration;
+use std::{f32::consts::TAU, time::Duration};
```

Comment:
:eyes:

Reply:
I'm using `TAU` in the stress tests as they don't serve as actual examples (i.e. They're not minimal examples of how to do something with Bevy in an idiomatic way). Instead, They are tools used by engine contributors to test performance.

But those are my thoughts. I'm not 100% sure how others think about it.

Reply:
I'm a TAU lover, but I think it's a controversial point.

Reply:
If I have to write `PI * 2.`, a part of my soul will die, but I will do it to get this merged :)